### PR TITLE
#36:   QR Code Generator

### PR DESCRIPTION
**Pull request by Arisha2902**
I have made the "QR Code Generator", which is responsive as well as interactive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,4 +122,26 @@ Python Scripts for solving daily life problems in windows (recommended)
 * In any case, I have added the **base** model in the **models** folder for you to use it directly without using gb's of your data for the bigger models.
 * The **base** model is kept in parts *lesser than 25MB* so that it can be uploaded to github, you need to extract it using ***7zip*** or ***winrar*** to use it. Extract using the **extract here** option.
 * The subtitle generation is done locally and the video is not uploaded to any server as per my understanding.
-* **NOTE:** ***If you face any issues with the path of videos while running the tree script or the other two scripts then try moving the working folder closer to the root directory of the drive. It might solve the issue.***
\ No newline at end of file
+
```
Comment (PreyumKr) [inline README.md:133]: The Readme.md changes show that you added few lines from above again below such as the simple finance tracker and the note for the subtitle generator is missing in the latest readme that is pushed. Maybe try fetching the latest readme.md and edit that one and change upon it once. If the lines still show up like this, I'll make a copy on my end and fix the issue.

**Review by PreyumKr — CHANGES_REQUESTED** [review]
Please re-commit the outer readme.md by editing the latest readme file from the repo. It is showing weird changes in the readme file. It is showing simple finance tracker below the subtitle generator and the note in the subtitle generator instructions is gone. If its actually done in your end's readme.md then please take the latest readme and just add the QR one in the lexical order of the titles, keeping others the same.

File: README.md (hunk 1 above)
Comment (Arisha2902) [reply README.md:133]: Sure let me try to recorrect the readme

Comment (Arisha2902) [reply README.md:133]: I have done changes in readme,i have added "qr code generater" in lexial order.
I have done this changes after pulling the last changes in readme.

**Review by PreyumKr — APPROVED** [review]
Still the note looks missing well.

**Pull request merged**